src/frontend/js/components/SearchFilterGroupModal/index.tsx

```diff
@@ -9,6 +9,25 @@ import { FacetedFilterDefinition, FilterValue } from 'types/filters';
 import { Nullable } from 'utils/types';
 import { useAsyncEffect } from 'utils/useAsyncEffect';
 
+/**
+ * Flex children in a fixed/absolute container breaks Safari.
+ * Use a specific, admittedly hacky media query to target it with a special layout
+ * that solves the issue on that browser.
+ */
+const safariFixStyles = `
```

Comment:
Not a fan to add a `style` tag anywhere on the page.

You could add these styles within `body.has-search-filter-group-modal` in `SearchFilterGroupModal/_styles.scss ` right?

Wow never this hacky media query to target Safari! Just to my understanding, where did you find this?

Reply:
I took this one from this blog post (https://www.ryadel.com/en/css3-media-query-target-only-ie-ie6-ie11-firefox-chrome-safari-edge/) which I've used as a reference in the past.

The issue is that `_styles.scss` does not work as this is not valid SASS (although is does work in CSS). It is in that sense I said it's a hack. I don't really have a better solution to offer.

The solution I propose for Safari is a worse experience in my opinion (as scrolling only through the list makes the UI less clear), which I'd rather not force on all other browsers. On the other hand, I wrestled with it a little bit this morning and could not find a way to make it work as intended on Safari. It seems like the sizing of flex children inside the `fixed` element is fundamentally broken.

Reply:
I think everything else is done now, we just have this one left to decide on.

If you want to try your hand to a better fix than what I offer in this PR,  there might be something I missed!

Reply:
This snippet works on my side. Just use a valid a media query (`@media all`)
```scss
body.has-search-filter-group-modal {
  overflow: hidden;

  @media not all and (min-resolution: 0.001dpcm) {
    @media all {
      .search-filter-group-modal {
        overflow: hidden;
      }
  
      .search-filter-group-modal__form__values {
        overflow: auto;
      }
    }
  }
}
```

Reply:
Great! I tried it on Edge and it works there too.

I changed it to your suggestion.